[2025-02-04T21:41:50.358+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-04T21:41:50.372+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_que_faire_a_paris.extract_data manual__2025-02-04T21:41:48.646826+00:00 [queued]>
[2025-02-04T21:41:50.382+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_que_faire_a_paris.extract_data manual__2025-02-04T21:41:48.646826+00:00 [queued]>
[2025-02-04T21:41:50.382+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-04T21:41:50.511+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_data> on 2025-02-04 21:41:48.646826+00:00
[2025-02-04T21:41:50.516+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=88) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-04T21:41:50.517+0000] {standard_task_runner.py:72} INFO - Started process 90 to run task
[2025-02-04T21:41:50.517+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_que_faire_a_paris', 'extract_data', 'manual__2025-02-04T21:41:48.646826+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/dag_que_faire_a_paris.py', '--cfg-path', '/tmp/tmpzd3zlnpd']
[2025-02-04T21:41:50.518+0000] {standard_task_runner.py:105} INFO - Job 50: Subtask extract_data
[2025-02-04T21:41:50.548+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_que_faire_a_paris.extract_data manual__2025-02-04T21:41:48.646826+00:00 [running]> on host 2023d4c64a22
[2025-02-04T21:41:50.593+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_que_faire_a_paris' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-04T21:41:48.646826+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-04T21:41:48.646826+00:00'
[2025-02-04T21:41:50.594+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-04T21:41:53.168+0000] {dag_que_faire_a_paris.py:34} INFO - Extraction completed successfully!
[2025-02-04T21:41:53.173+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-04T21:41:53.179+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-04T21:41:53.179+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_que_faire_a_paris, task_id=extract_data, run_id=manual__2025-02-04T21:41:48.646826+00:00, execution_date=20250204T214148, start_date=20250204T214150, end_date=20250204T214153
[2025-02-04T21:41:53.221+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-04T21:41:53.245+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-04T21:41:53.248+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
